Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM alpine:3
-RUN apk add --update --no-cache bash ca-certificates curl git jq openssh
+RUN apk add --update --no-cache bash ca-certificates curl git jq openssh tar lastversion
```

Comment:
tar - because the busybox tar doesn't support wildcard
lastversion - because it provides an easy way to do semver comparisons (for the alpha plugins syntax)